### DISCUSSION TITLE
Find out panel ids of a dashboard with repeated rows via API

### DISCUSSION THREAD

**Discussion by mjsaarin** [grafana/grafana#41790] [Dashboards, Panels & Visualizations]
Hello,

I have a dashboard that has repeated rows. Each row has three panels. Now, I would like to be able to render and share an image of an individual panel. All this would be run from cron. For that, I will need panel id. With Grafana 7x. I could pull the JSON model and collect ids from there. With Grafana 8.x this is no longer possible and my request for the feature was bluntly discarded. 

Is there a way to query for the panel ids via API in Grafana 8.x? Of course I could run an addtional virtual server with Grafana 7.x for particular dashboards only. I'd rather avoid that.

Cheers,

Matti

**Comment by torkelo** [comment]
you can use puppeter (chrome/browser) automation to load a dashboard, and scan for the panels generated by the dynamic repeat process

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
